bugbug/models/backout.py

```diff
@@ -9,13 +9,13 @@
 from sklearn.feature_extraction import DictVectorizer
 from sklearn.pipeline import Pipeline
 
-from bugbug import commit_features, feature_cleanup, repository
+from bugbug import bug_features, commit_features, feature_cleanup, repository
 from bugbug.model import CommitModel
 
 
 class BackoutModel(CommitModel):
-    def __init__(self, lemmatization=False):
-        CommitModel.__init__(self, lemmatization)
+    def __init__(self, lemmatization=False, bug_data=False):
+        CommitModel.__init__(self, lemmatization, bug_data)
```

Comment:
When bug_data is True, we should only get the commits for which we have bug_data, so in get_labels we should have the same date check as in bug_retriever

Reply:
Done.

Reply:
I think there are no dependencies missing, you are not using any new package.